### ISSUE TITLE
Issues with larger cluster and overstate system

### ISSUE THREAD

**Issue by abemusic** [saltstack/salt#9297]
Until recently we had only been running smaller clusters while developing stackd.io, but last week we had our first major internal adopter that started using it to manage the clusters we need internally for our large-scale integration testing of our company's product. The first "large" cluster was a 50-node Hadoop/HBase cluster along with our own proprietary software and we immediately encountered issue with executing an overstate file that previously worked for smaller test clusters.

I'll try to provide all the information I can, but if I miss something or there are questions please let me know. I would love to figure this out so we can move forward:

```bash

```

**Comment by jfindlay** [comment]
@abemusic, is this issue still valid?  The orchestrate runner (previously the state.sls runner) has deprecated the overstate system.

**Comment by abemusic** [comment]
@jfindlay - I left that company in Aug2014 so I can't be totally sure where we were at with this particular problem. IIRC, we were waiting on the new runner to help with orchestration issues, but nobody could really give us an examples or guidance on getting it stood up, so I wrote an orchestration runner that we used from that point forward. It worked well enough to keep us moving with our internal projects and I'm assuming they are still using it as I was the only one at the time that was interested in digging into Salt and not just staying on the ops side.
